core/src/main/java/software/amazon/awssdk/http/pipeline/stages/AsyncRetryableStage.java

```diff
@@ -126,19 +118,17 @@ private static int parseClockSkewOffset(HttpResponse httpResponse) {
         private final SdkHttpFullRequest request;
         private final RequestExecutionContext context;
         private final ProgressListener progressListener;
-        private final AwsRequestMetrics awsRequestMetrics;
 
         private Optional<SdkBaseException> retriedException;
         private RetryPolicyContext retryPolicyContext;
-        private int requestCount;
+        private int requestCount = 0;
```

Comment:
default value is anyways zero. Is this new standard we follow?

Reply:
It's just something that made me pause and think about it because we were incrementing a number that hadn't been set. I added "= 0" to keep me from having to think in the future.